src/Hedgehog/Plutus/Adjunction.hs

```diff
@@ -32,8 +32,10 @@ instance Category Adjunction where
 -- | Verify the adjunction of a layer.
 adjunctionTest ::
   forall (m :: Type -> Type) (a :: Type) (b :: Type).
-  (Hedgehog.MonadTest m, Eq a, Show a) =>
+  (Hedgehog.MonadTest m, Eq a, Show a, Show b) =>
   Adjunction b a ->
   a ->
   m ()
-adjunctionTest (Adjunction lower raise) a = a === raise (lower a)
+adjunctionTest (Adjunction lower raise) a = do
+  annotateShow (lower a)
```

Comment:
Why?

Reply:
It was helpful for debugging the time tests and I expect it will be helpful to anyone else debugging adjunction tests.

Reply:
That makes sense.